apps/blaze-dashboard/src/load-config.js

```diff
@@ -0,0 +1,21 @@
+// The JSON is filtered by `apps/blaze-dashboard/filter-json-config-loader.js`.
+import productionConfig from '../../../config/production.json';
+
+// Set is_running_in_jetpack_site to true if not specified (undefined or null).
+productionConfig.features.is_running_in_jetpack_site =
+	window.configData.features.is_running_in_jetpack_site ?? true;
+
+// The option enables loading of the whole translation file, and could be optimized by setting it to `true`, which needs the translation chunks in place.
+// @see https://github.com/Automattic/wp-calypso/blob/trunk/docs/translation-chunks.md
+productionConfig.features[ 'use-translation-chunks' ] = false;
+
+// Override the redesign feature
+productionConfig.features[ 'promote-post/redesign-i2' ] = true;
+productionConfig.dsp_widget_js_src = 'https://widgets.wp.com/promote-v2/widget.js';
```

Comment:
Are we forcing to use v2 of campaign dashboard and v2 of blaze widget here? I don't know much about standalone apps in Calypso. Is the current version a standalone app? We are also trying to test the v2 widget redesign with Jetpack, but I'm concerned that with forcing to use `/promote-v2/widget.js` here, we'll expose v2 to the users before releasing it.

Reply:
We are forcing both of them here because right now, both are using the same feature flag, and by the time we merge this PR there was an error that if you used the `promote/widget.js` entry point, with the `isV2` flag in true it showed you an mix version of the widget (new redesign HTML but with old CSS). 

This app is not being used anywhere yet, we plan to release the first version when we finish all the redesign tasks and enable it for users in Jetpack after all the testing is done. 

I just tested it and the bug I mentioned before appears to be fixed, so we can make a PR removing this line, which will force the app to take the config value from production instead (that is the v1 entry point).